src/engage/enrollments/enrollment.action.ts

```diff
@@ -0,0 +1,107 @@
+import {ZapierItem} from '../../types'
+import {createEnrollment, stopAllEnrollments} from '../common'
+import {createActionDetails} from '../../utils/operations'
+import {userSearches, userTriggers} from '../../users/keys'
+import {enrollmentActions, sequenceSearches, sequenceTriggers} from '../keys'
+import EnrollmentResource from './enrollment.resource'
+import {leadSearches, leadTriggers} from '../../lead/keys'
+
+export const CreateEnrollmentAction: ZapierItem = {
+    key: enrollmentActions.createEnrollmentAction,
+    noun: 'Enrollment',
+    display: {
+        label: 'Create sequence enrolment',
+        description: 'Creates a sequence enrolment for a given lead. Requires Reach add-on.'
+    },
+    operation: {
+        resource: EnrollmentResource.key,
+        inputFields: [
+            {
+                key: 'sequence_id',
+                label: 'Sequence',
+                type: 'integer',
+                required: true,
+                dynamic: `${sequenceTriggers.sequenceListDropdown}.id.name`,
+                search: `${sequenceSearches.sequenceSearch}.id`,
+            },
+            {
+                key: 'sell_resource',
+                label: 'Sell Resource',
+                children: [
+                    {
+                        key: 'resource_id',
+                        label: 'Object ID',
+                        helpText: 'Zendesk Sell Lead that will be enrolled.',
+                        required: true,
+                        type: 'integer',
+                        dynamic: `${leadTriggers.leadListDropdown}.id.name`,
+                        search: `${leadSearches.leadSearchOrCreate}.id`
+
+                    },
+                    {
+                        key: 'resource_type',
+                        label: 'Object Type',
+                        choices: ['lead'],
+                        default: 'lead',
+                        type: 'string',
+                        required: true
+                    }
+                ]
+            },
+            {
+                key: 'actor_id',
+                label: 'Sequence Owner ID',
+                helpText: 'That will be the user that emails will be sent by and tasks will be created for.',
```

Comment:
I'm no copy expert (so you can ignore this comment), but this could be shorter ;)
`User that emails will be sent by and tasks will be created for.`

cc @piotrplaneta

Reply:
leaving as is